lecture_2/homework/app/models/stronghold.rb

```diff
@@ -0,0 +1,3 @@
+class Stronghold < ApplicationRecord
+  has_many :warriors, as: :defensible
```

Comment:
Masz `null: false` w migracjach, ale nie masz walidacji. To powoduje wyrzucenie błędu z bazy danych i 500 w aplikacji 😢